internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -616,3 +616,33 @@ func convertTopic(t manifest.Topic) (*template.Topics, error) {
 		AllowedWorkers: t.AllowedWorkers,
 	}, nil
 }
+
+func convertSubscribe(s *manifest.SubscribeConfig) (*template.SubscribeOpts, error) {
+	if s == nil || s.Topics == nil {
+		return nil, nil
+	}
+
+	subscriptions := template.SubscribeOpts{}
```

Comment:
```suggestion
	var subscriptions template.SubscribeOpts
```
This also initializes `subscriptions` to zero value. We prefer this over the short declaration throughout the codebase if we don't need any special initialization as the latter is a "declare-and-assign" operation and involves type inference